client/network/src/protocol.rs

```diff
@@ -1032,13 +1032,14 @@ impl<B: BlockT, H: ExHashT> Protocol<B, H> {
 
 	/// Registers a new notifications protocol.
 	///
-	/// You are very strongly encouraged to call this method very early on. Any connection open
-	/// will retain the protocols that were registered then, and not any new one.
-	pub fn register_notifications_protocol(
-		&mut self,
+	/// While registering a protocol while we already have open connections is discouraged, we
```

Comment:
```suggestion
	/// Registering a protocol while we already have open connections is discouraged, we
```

Reply:
I don't see this as wrong. It's a synonym of "though".